prometheus/src/cluster_metrics.rs

```diff
@@ -52,5 +81,60 @@ pub fn write_cluster_metrics<W: io::Write>(
         },
     )?;
 
+    // Vote accounts information
+    for vote_account in vote_accounts.iter() {
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_last_vote_slot",
+                help:
+                    "The voted-on slot of the validator's last vote that got included in the chain",
+                type_: "gauge",
+                metrics: banks_with_commitments.for_each_commitment(|bank| {
+                    let vote_info = get_vote_state(bank, vote_account)?;
+                    Some(
+                        Metric::new(vote_info.last_vote)
+                            .with_label("identity_account", identity_pubkey.to_string())
```

Comment:
This identity pubkey is defined at the beginning of the scope, it is _our_ identity (of the running process), not the one that goes with the vote account! I think we can get the identity from `VoteState::node_pubkey`.

Reply:
Good catch!